src/content/docs/browser/new-relic-browser/configuration/view-browser-apps-alert-information.mdx

```diff
@@ -59,5 +59,5 @@ If an alert condition has [thresholds](/docs/alerts/new-relic-alerts-beta/gettin
 3. Optional: To view or update the alert policies, conditions, or thresholds that triggered the violations, select the corresponding link.
 
 <Callout variant="tip">
-  You can also view Warning and Critical violations from the New Relic One Alerts & AI UI for [specific incidents](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/explore-incident-history) or for [incidents and events across all products](/docs/alerts/new-relic-alerts-beta/reviewing-events/review-events-across-products). Alerts sends [alert notifications](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/notification-channels-controlling-where-send-alerts) only for Critical (red) violations.
+  You can also view Warning and Critical violations from the Alerts & AI UI for [specific incidents](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/explore-incident-history) or for [incidents and events across all products](/docs/alerts/new-relic-alerts-beta/reviewing-events/review-events-across-products). Alerts sends [alert notifications](/docs/alerts/new-relic-alerts-beta/managing-notification-channels/notification-channels-controlling-where-send-alerts) only for Critical (red) violations.
```

Comment:
@akristen, I wonder if "Warning" and "Critical" can be lower case. You my check with @paperclypse.

Reply:
Yes, I think they should be lowercase.